### ISSUE TITLE
🐞 Bug Report: Fatal errors when saving "Site Settings". PHP 8.0 (🔃 Has PR)

### ISSUE THREAD

**Issue by gerasimovdaniel** [elementor/elementor#20062]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hi! I faced some issues with Elementor + Elementor PRO + PHP 8.0. Saving "Site Setting" causes PHP Fatal errors.

```
Uncaught TypeError: Cannot access offset of type string on string in /wp-content/plugins/elementor-pro/plugin.php:350
Stack trace:
/wp-includes/class-wp-hook.php(309): ElementorPro\Plugin->get_responsive_stylesheet_templates('...')
/wp-includes/plugin.php(235): WP_Hook->apply_filters(NULL, Array)
/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(357): apply_filters_ref_array('elementor/core/...', Array)
/wp-content/plugins/elementor/core/breakpoints/manager.php(533): Elementor\Modules\DevTools\Deprecation->apply_deprecated_filter('elementor/core/...', Array, '3.2.0', 'elementor/core/...')
/wp-content/plugins/elementor/core/breakpoints/manager.php(420): Elementor\Core\Breakpoints\Manager::get_stylesheet_templates()
/wp-content/plugins/elementor/core/kits/documents/tabs/settings-layout.php(292): Elementor\Core\Breakpoints\Manager::compile_stylesheet_templates()
/wp-content/plugins/elementor/core/kits/documents/kit.php(98): Elementor\Core\Kits\Documents\Tabs\Settings_Layout->on_save(Array)
[internal function]: Elementor\Core\Documents_Manager->ajax_save(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
/wp-includes/class-wp-hook.php(307): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters('', Array)
/wp-includes/plugin.php(476): WP_Hook->do_action(Array)
/wp-admin/admin-ajax.php(187): do_action('wp_ajax_element...')
{main} thrown in /wp-content/plugins/elementor-pro/plugin.php on line 350
```

### Steps to reproduce

1) Open Elementor "Site Settings"
2) Add global color
3) Save

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

WordPress - 6.0.2
Elementor - 3.7.8
Elementor Pro - 3.7.3
PHP - 8.0.24

**Comment by nicholaszein** [comment]
Hello @gerasimovdaniel! 

Thank you for submitting your issue, and PR! 🙏 

Please allow us some time to evaluate it, and if everything passes we would be glad to approve it in the future.

Best regards

**Comment by arekuki** [comment]
+

**Comment by StyleShit** [comment]
Resolved in #20063, #20327.

Thanks!

**Comment by nicholaszein** [comment]
Hello @gerasimovdaniel!

Your PR has been merged, and the changes have been scheduled to be applied in Elementor 3.9.0.

Thank you for contributing! 💪

**Comment by nicholaszein** [comment]
Hello there @gerasimovdaniel!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor **v3.8.1**! 🥳 

✔️ Please feel free to [check it out](https://wordpress.org/plugins/elementor/#developers) and update your plugin to the new version!

Cheers 🥂
